### PR TITLE
Move to GitHub actions.

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "@nationalarchives/file-information",
-  "version": "1.0.311",
+  "version": "1.0.312",
```
Comment (TomJKing) [inline package-lock.json:3]: `package.json` is showing version as `1.0.313`?

**Review by TomJKing — COMMENTED** [review]
Looks good. One minor comment.

File: package-lock.json (hunk 1 above)
Comment (MancunianSam) [reply package-lock.json:3]: I've changed package.json to be 312. It doesn't need to be 313.

**Review by TomJKing — DISMISSED** [review]
Looks good. Thanks

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks.

**Pull request merged**